addons/source-python/packages/source-python/entities/_base.py

```diff
@@ -772,7 +910,10 @@ def set_property_vector(self, name, value):
         :param Vector value:
             The value to set.
         """
-        self._set_property(name, 'Vector', value)
+        try:
+            self.set_network_property_vector(name, value)
+        except ValueError:
+            self.set_datamap_property_vector(name, value)
```

Comment:
Shouldn't we do the same with ``Edict`` and then remove ``_get_property`` and ``_set_property``. That seems to be the last one that uses the slow variant by iterating over the server classes and datamaps.

Reply:
Ideally, yes. But `BaseEntity.get_<network/datamap>_property_edict` are not implemented and as far as dumping goes, there isn't any properties that are edicts (at least, on OB games) to implement them and base tests upon. So they were left as is, although I think they could probably be removed. Not even sure they are `edict_t *` as the current implementation assume, and they could well be anything that require to be proxied as far as I know.